### PR TITLE
dev-python/plotly: version bump to 4.10.0

### PR DESCRIPTION
**Pull request by epsilon-0**
Closes: https://bugs.gentoo.org/641650

Also adds new package maintainer

Package-Manager: Portage-3.0.7, Repoman-3.0.1
Signed-off-by: Aisha Tammy <gentoo@aisha.cc>

### CODE REVIEW

File: dev-python/plotly/plotly-4.10.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# blocker dev-python/retrying
```
Comment (thesamesam) [inline dev-python/plotly/plotly-4.10.0.ebuild:6]: Got a bug for it? ;)

File: dev-python/plotly/plotly-4.10.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/retrying[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}"
```
Comment (thesamesam) [inline dev-python/plotly/plotly-4.10.0.ebuild:23]: What for? Usually we don't need to bother with this, just at runtime, but maybe they are needed?

File: dev-python/plotly/plotly-4.10.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# blocker dev-python/retrying
+PYTHON_COMPAT=( python3_{6..7} )
+
+inherit distutils-r1 virtualx
```
Comment (thesamesam) [inline dev-python/plotly/plotly-4.10.0.ebuild:9]: What's virtual here for - tests? Do we need distutils_enable_tests or similar?

File: dev-python/plotly/plotly-4.10.0.ebuild (hunk 1 above)
Comment (epsilon-0) [reply dev-python/plotly/plotly-4.10.0.ebuild:6]: `retrying ` is `python3_{6,7}` only hence I can't add `8,9` as of now.

File: dev-python/plotly/plotly-4.10.0.ebuild (hunk 3 above)
Comment (epsilon-0) [reply dev-python/plotly/plotly-4.10.0.ebuild:9]: Ah, a remnant from previous, will remove.

File: dev-python/plotly/plotly-4.10.0.ebuild (hunk 2 above)
Comment (epsilon-0) [reply dev-python/plotly/plotly-4.10.0.ebuild:23]: Unsure how to do this without a lot of pain.
Don't see a harm in letting it stay.

Comment (epsilon-0) [reply dev-python/plotly/plotly-4.10.0.ebuild]: done

**Comment by epsilon-0** [comment]
There are no tests distributed with the package.

**Comment by epsilon-0** [comment]
CI is running on an older version of the tree, which hasn't yet update the retrying to 3_{8,9}.
It should be safe to ignore.

File: dev-python/chart-studio/chart-studio-1.1.0.ebuild [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+
+DESCRIPTION="Browser-based graphing library for Python"
+HOMEPAGE="https://plot.ly/python/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="${PYTHON_DEPS}
```
Comment (SoapGentoo) [inline dev-python/chart-studio/chart-studio-1.1.0.ebuild:18]: `${PYTHON_DEPS}` redundant, `distutils-r1` injects it implicitly

File: dev-python/chart-studio/chart-studio-1.1.0.ebuild [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+
+inherit distutils-r1
+
+DESCRIPTION="Browser-based graphing library for Python"
+HOMEPAGE="https://plot.ly/python/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
```
Comment (SoapGentoo) [inline dev-python/chart-studio/chart-studio-1.1.0.ebuild:16]: always use the following block format:
```
DESCRIPTION="Browser-based graphing library for Python"
HOMEPAGE="https://plot.ly/python/"
SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"

LICENSE="MIT"
SLOT="0"
KEYWORDS="~amd64 ~x86"
```
i.e. there's a free line between `SRC_URI` and `LICENSE`

File: dev-python/plotly-geo/plotly-geo-1.0.0.ebuild [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..9} )
+
+inherit distutils-r1
+
+DESCRIPTION="Browser-based graphing library for Python"
+HOMEPAGE="https://plot.ly/python/"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+KEYWORDS="~amd64 ~x86"
```
Comment (SoapGentoo) [inline dev-python/plotly-geo/plotly-geo-1.0.0.ebuild:13]: same

File: dev-python/plotly/plotly-4.10.0.ebuild [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="${PYTHON_DEPS}
+	dev-python/retrying[${PYTHON_USEDEP}]
+	dev-python/six[${PYTHON_USEDEP}]
+"
+
+python_install_all() {
```
Comment (SoapGentoo) [inline dev-python/plotly/plotly-4.10.0.ebuild:23]: any tests here? all things equal, tests are more important in python packages than C/C++ packages

File: dev-python/plotly/plotly-4.10.0.ebuild [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+	distutils-r1_python_install_all
+
+	mkdir -p "${ED}"/etc/
```
Comment (SoapGentoo) [inline dev-python/plotly/plotly-4.10.0.ebuild:26]: missing `|| die`

**Review by SoapGentoo — CHANGES_REQUESTED** [review]

File: dev-python/chart-studio/chart-studio-1.1.0.ebuild (hunk 4 above)
Comment (epsilon-0) [reply dev-python/chart-studio/chart-studio-1.1.0.ebuild]: removed.

File: dev-python/chart-studio/chart-studio-1.1.0.ebuild (hunk 5 above)
Comment (epsilon-0) [reply dev-python/chart-studio/chart-studio-1.1.0.ebuild:15]: done

File: dev-python/plotly-geo/plotly-geo-1.0.0.ebuild (hunk 6 above)
Comment (epsilon-0) [reply dev-python/plotly-geo/plotly-geo-1.0.0.ebuild]: done

File: dev-python/plotly/plotly-4.10.0.ebuild (hunk 7 above)
Comment (epsilon-0) [reply dev-python/plotly/plotly-4.10.0.ebuild:23]: for sake of others who might read.
these packages don't contain tests, they are made from a single source tarball which contains 6 different packages (broken up later for pypi releases), so we can't use github releases as well.

File: dev-python/plotly/plotly-4.10.0.ebuild (hunk 8 above)
Comment (epsilon-0) [reply dev-python/plotly/plotly-4.10.0.ebuild]: done

**Pull request closed without merging**